### PR TITLE
AB#7900: fix translations

### PR DESCRIPTION
**Pull request by josejibin**
Continuation from the [PR](https://github.com/shift72/core-template/pull/336)

* Fix the suggestions from the [PR](https://github.com/shift72/core-template/pull/336)
* Remove the whitespace from mustaches variables

### CODE REVIEW

**Review by PeculiarGoat — APPROVED** [review]
Nice work!  Sorry I missed this on the previous PR... reviewing translations is mind-numbing!

**Review by stajs — APPROVED** [review]
Thanks for tidying those up! Only had one question but not a blocker for me.

File: site/ca_ES.all.json [hunk 1]
```diff
@@ -1181,15 +1181,15 @@
     "other": "Preu"
   },
   "shopping_price_plan_note": {
-    "one": "Es {{ .Interval }} cada .Interval després que finalitzi la prova gratuïta de 24 hores.",
-    "other": "Es {{ .Interval }} cada .Interval després que finalitzi la prova gratuïta de {{ .Count }}"
+    "one": "Es {{.Interval}} cada .Interval després que finalitzi la prova gratuïta de 24 hores.",
```
Comment (l0ud0gg) [inline site/ca_ES.all.json:1184]: this has .Interval in there twice, with the second not in moustaches. not sure if thats intended or not?

File: site/da_DK.all.json [hunk 2]
```diff
@@ -1126,18 +1126,18 @@
...
-    "other": "Dit abonnementskøb blev gennemført. Du abonnerer nu på {{ .Title }} ."
+    "other": "Dit abonnementskøb blev gennemført. Du abonnerer nu på {{.Title}} ."
   },
   "shopping_complete_subscription_browse": {
     "other": "Du er velkommen til at <a href=\"/\">browse rundt</a> på siden, eller <a href=\"/search.html\">søg</a> efter noget specifikt."
   },
   "shopping_info_plan_offer": {
-    "one": "{{ .Interval }} automatisk hvert .Interval",
-    "other": "{{ .Count }} automatisk hver {{ .Interval }}"
+    "one": "{{.Interval}} automatisk hvert .Interval",
```
Comment (l0ud0gg) [inline site/da_DK.all.json:1135]: this has .Interval in there twice, with the second not in moustaches. not sure if thats intended or not?

File: site/et_ET.all.json [hunk 3]
```diff
@@ -1013,15 +1013,15 @@
     "other": "Hind"
   },
   "shopping_price_plan_note": {
-    "one": "{{ .Interval }} iga .intervalli järel pärast 24-tunnise tasuta prooviperioodi lõppu.",
-    "other": "{{ .Interval }} iga .intervalli järel pärast {{ .Count }} päeva tasuta prooviperioodi lõppu."
+    "one": "{{.Interval}} iga .intervalli järel pärast 24-tunnise tasuta prooviperioodi lõppu.",
```
Comment (l0ud0gg) [inline site/et_ET.all.json:1016]: is .intervalli supposed to be here?

File: site/et_ET.all.json [hunk 4]
```diff
@@ -1013,15 +1013,15 @@
...
+    "other": "{{.Interval}} iga .intervalli järel pärast {{.Count}} päeva tasuta prooviperioodi lõppu."
   },
   "shopping_price_plan_note_without_trial": {
-    "other": "{{ .Interval }} iga .Intervall ."
+    "other": "{{.Interval}} iga .Intervall ."
```
Comment (l0ud0gg) [inline site/et_ET.all.json:1020]: is .Intervall supposed to be here?

File: site/no_NO.all.json [hunk 5]
```diff
@@ -1021,18 +1021,18 @@
...
-    "other": "Abonnementskjøpet var vellykket. Du abonnerer nå på {{ .Title }} ."
+    "other": "Abonnementskjøpet var vellykket. Du abonnerer nå på {{.Title}} ."
   },
   "shopping_complete_subscription_browse": {
     "other": "<a href=\"/\">Bla deg gjerne rundt</a> på siden, eller <a href=\"/search.html\">søk</a> etter noe spesifikt."
   },
   "shopping_info_plan_offer": {
-    "one": "{{ .Interval }} automatisk hver .Interval",
-    "other": "{{ .Count }} automatisk hver {{ .Interval }}"
+    "one": "{{.Interval}} automatisk hver .Interval",
```
Comment (l0ud0gg) [inline site/no_NO.all.json:1030]: Is there supposed to be a second .Interval here?

File: site/pl_PL.all.json [hunk 6]
```diff
@@ -912,18 +912,18 @@
...
   "shopping_info_plan_offer": {
-    "one": "Automatycznie odnawia co {{ .Interval }}",
-    "other": "Automatycznie odnawia co {{ .Count }} {{ .Interval }}"
+    "one": "Automatycznie odnawia co {{.Interval}}",
+    "other": "Automatycznie odnawia co {{.Count}} {{.Interval}}"
   },
   "shopping_info_trial_period_offer": {
     "one": "Wypróbuj bezpłatny 24-godzinny okres próbny już teraz!",
-    "other": "Wypróbuj bezpłatną {{ .Count }} próbną programu .Count!"
+    "other": "Wypróbuj bezpłatną {{.Count}} próbną programu .Count!"
```
Comment (l0ud0gg) [inline site/pl_PL.all.json:926]: is there supposed to be a second .Count in here?

**Review by l0ud0gg — CHANGES_REQUESTED** [review]
Mostly good. left a few comments around duplicate .Interval's etc.

File: site/it_IT.all.json [hunk 7]
```diff
@@ -931,18 +931,18 @@
     "other": "Completare"
   },
   "shopping_complete_subscription": {
-    "other": "L&#39;acquisto dell&#39;abbonamento è andato a buon fine. Ora sei iscritto a {{ .Title }} ."
+    "other": "L&#39;acquisto dell&#39;abbonamento è andato a buon fine. Ora sei iscritto a {{.Title}} ."
```
[the remark this answers is not in the document]
Comment (stajs) [reply site/it_IT.all.json]: Nope! It was only if you were going to rely on the HTML entities.

File: site/et_ET.all.json [hunk 8]
```diff
@@ -1013,11 +1013,11 @@
     "other": "Hind"
   },
   "shopping_price_plan_note": {
-    "one": "{{.Interval}} iga .intervalli järel pärast 24-tunnise tasuta prooviperioodi lõppu.",
+    "one": "Tasu võetakse iga {{.Interval}} järel pärast 24-tunnise tasuta prooviperioodi lõppu.",
     "other": "{{.Interval}} iga .intervalli järel pärast {{.Count}} päeva tasuta prooviperioodi lõppu."
```
Comment (l0ud0gg) [inline site/et_ET.all.json:1017]: sorry one more .intervallii here in the :other

**Review by l0ud0gg — APPROVED** [review]
one nit pick comment. otherwise all good 👍

**Pull request merged**